blinky/__init__.py

```
@@ -1,256 +1,130 @@
-    # Configuration (from environment variables)
-    DISCORD_BOT_TOKEN = os.getenv("DISCORD_BOT_TOKEN")
-    GOOGLE_API_KEY = os.getenv("GOOGLE_API_KEY")
 
-    # Error checks for config
-    if not DISCORD_BOT_TOKEN:
-        raise ValueError("DISCORD_BOT_TOKEN environment variable not set.")
-    if not GOOGLE_API_KEY:
-        raise ValueError("GOOGLE_API_KEY environment variable not set.")
+# Error checks for config
+if not DISCORD_BOT_TOKEN:
+    raise ValueError("DISCORD_BOT_TOKEN environment variable not set.")
```

Ruff (0.14.4)

Avoid specifying long messages outside the exception class

(TRY003)